app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -40,6 +40,10 @@ def self.default_blacklisted_event_names
     )
   end
 
+  def self.metrics_collect_queue_name
+    MetricsCollectorWorker.default_queue_name
```

Comment:
Will that cause any issues with providers that inherit from `ManageIQ::Providers::BaseManager` but don't have a `self::MetricsCollectorWorker`?  The only one I could find right now is `ManageIQ::Providers::Microsoft::InfraManager`